apps/mobile/src/screens/Profile.tsx

```diff
@@ -1,39 +1,170 @@
+    { pfpSrc: require('../assets/images/mocks/ape.png') },
+    { pfpSrc: require('../assets/images/mocks/ape3.png') },
+    { pfpSrc: require('../assets/images/mocks/ape.png') },
+]
+const mockSbtGallery = [
+    { pfpSrc: require('../assets/images/mocks/sbt.png') },
+    { pfpSrc: require('../assets/images/mocks/sbt.png') },
+]
+
+enum EGallery {
+    NFTS = 'NFTs',
+    SBTS = 'SBTs',
+}
+
 export const Profile = () => {
-    const { name, description, pfp, twitter } = mockProfile
+    const { name, description, pfp, location, occupation, organization, whatCan, wantMeet } = mockProfile
+
+    const [selectedGalleryTab, setSelectedGalleryTab] = useState(EGallery.NFTS)
+    const [isPreferedContact, setIsPreferedContact] = useState(false)
+    const [isFullView, setIsFullView] = useState(false)
+
+    const fullProfileTitle = isFullView ? 'Done' : 'Full Profile'
+
     const navigation = useNavigation()
+    const address = '0x32...2sak3' // TODO get address from wallet
+
+    const toggleSwitch = () => setIsPreferedContact(previousState => !previousState)
 
     function handleEditLink() {
         navigation.navigate('Create')
     }
 
+    const renderGalleryButton = (galleryName: EGallery) => (
+        <TouchableOpacity
+            onPress={() => setSelectedGalleryTab(galleryName)}
+            style={{
+                width: '50%',
+            }}>
+            <NoteMono
+                style={{
+                    paddingTop: 12,
+                    textAlign: 'center',
+                    height: 42,
+                    backgroundColor: selectedGalleryTab === galleryName ? '#353844' : '#6d6d6f',
+                }}>
+                {galleryName}
+            </NoteMono>
+        </TouchableOpacity>
+    )
+    const renderFullInfo = isFullView && (
+        <>
+            <Button label='Edit Profile' width={280} style={{ width: 260 }} onPress={handleEditLink} />
+            <InfoContainer title='Location' text={location} />
+            <InfoContainer title='Job Title' text={occupation} />
+            <InfoContainer title='Organization' text={organization} />
+            <InfoContainer title='Skills' text={whatCan} />
+            <InfoContainer title='Interested in meeting' text={wantMeet} />
+        </>
+    )
+
     return (
-        <ScrollView>
-            <Layout lightColor='#eee' darkColor='rgba(255,255,255,0.1)'>
-                <ContainerFlex>
-                    <SpaceEnd style={{ marginTop: 50 }}>
-                        <Button onPress={handleEditLink} title='Edit' />
-                    </SpaceEnd>
-                    <Avatar pfpCid={pfp} />
-                    <ContainerFlex mt='15px' mb='10px' p='0px'>
+        <Layout>
+            <ScrollView>
+                <CenteredContainer style={{ paddingHorizontal: 10 }}>
+                    <Image source={require('../assets/images/mocks/cover1.png')} />
+                    <View style={styles.backgroundContainer}>
+                        <Image
```

Comment:
This repeated image component does not look really well, maybe it is a better way to put it, but since we are migrating to nextjs let's keep as it is for now 👍

Reply:
Yeah I think 128 meant to be deleted, will make sure to not to add in in web version